.changeset/cool-dancers-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-react": patch
+---
+
+**Dialog**: Use `closedby="none | closerequest | any"` instead of `backdropClose` to align with [upcoming browser standard](https://chromestatus.com/feature/5097714453577728)
```

Comment:
```suggestion
**Dialog**: Remove `backdropClose`, and replace with `closedby="none | closerequest | any"` to align with [upcoming browser standard](https://chromestatus.com/feature/5097714453577728)
```
Just to get `backdropClose` earlier in the change so people are more likely to notice